### PR TITLE
EncryCommon, transitional deps removal, `Address` tags removal. (EC-372)

### PR DESCRIPTION
**Pull request by oskin1**

### CODE REVIEW

File: src/main/scala/encry/local/miner/Miner.scala [hunk 1]
```diff
@@ -2,10 +2,10 @@ package encry.local.miner
 
 import java.text.SimpleDateFormat
 import java.util.Date
+
```
Comment (kapinuss) [inline src/main/scala/encry/local/miner/Miner.scala]: ! here and below

File: src/main/scala/encry/modifiers/mempool/BaseTransaction.scala [hunk 2]
```diff
@@ -11,10 +11,12 @@ import encry.modifiers.state.box.{AssetBox, DataBox}
 import encry.modifiers.state.box.EncryBaseBox
 import encry.settings.{Algos, Constants}
 import io.circe.Encoder
+import org.encryfoundation.common.transaction.{Input, Proof}
 import org.encryfoundation.prismlang.compiler.CompiledContract
 import org.encryfoundation.prismlang.core.PConvertible
 import scorex.crypto.encode.Base16
 import scorex.crypto.hash.Digest32
+
```
Comment (Bromel777) [inline src/main/scala/encry/modifiers/mempool/BaseTransaction.scala]: empty line

File: src/main/scala/encry/modifiers/mempool/EncryTransaction.scala [hunk 3]
```diff
@@ -9,9 +9,11 @@ import encry.settings.Algos
 import encry.validation.{ModifierValidator, ValidationResult}
 import io.circe.syntax._
 import io.circe.{Decoder, Encoder, HCursor}
+import org.encryfoundation.common.transaction.{Input, InputSerializer, Proof, ProofSerializer}
 import org.encryfoundation.prismlang.core.Types
 import org.encryfoundation.prismlang.core.wrapped.{PObject, PValue}
 import scorex.crypto.hash.Digest32
+
```
Comment (Bromel777) [inline src/main/scala/encry/modifiers/mempool/EncryTransaction.scala]: also empty line

Comment (GRYE) [inline src/main/scala/encry/modifiers/mempool/EncryTransaction.scala]: blank line

File: src/test/scala/encry/modifiers/InstanceFactory.scala [hunk 4]
```diff
@@ -6,11 +6,13 @@ import encry.modifiers.state.box.Box.Amount
 import encry.modifiers.state.box.{AssetBox, EncryProposition}
 import encry.utils.{EncryGenerator, TestHelper}
 import encry.view.history.Height
+import org.encryfoundation.common.transaction.Input
 import org.encryfoundation.prismlang.compiler.CompiledContract
 import org.encryfoundation.prismlang.core.Ast.Expr
 import org.encryfoundation.prismlang.core.{Ast, Types}
 import scorex.crypto.authds.ADKey
 import scorex.utils.Random
+
```
Comment (jhnsmth) [inline src/test/scala/encry/modifiers/InstanceFactory.scala]: 🔥🔥🔥🔥🔥

**Pull request merged**